packages/core/database/encryption/README.md

```diff
@@ -0,0 +1,496 @@
+# Frigg Field-Level Encryption
+
+Database-agnostic field-level encryption for Frigg using Prisma Client Extensions and AWS KMS/AES.
+
+## Overview
+
+This module provides **transparent field-level encryption** for sensitive data in Frigg integrations. It works identically for MongoDB and PostgreSQL, using Prisma Client Extensions to automatically encrypt data on write and decrypt on read.
+
+### Key Features
+
+- ✅ **Database-agnostic**: Works with MongoDB, PostgreSQL, and future databases
+- ✅ **Transparent**: Repositories and use cases work with plain data
+- ✅ **Hexagonal architecture**: Clean separation of concerns
+- ✅ **AWS KMS support**: Enterprise-grade encryption with AWS Key Management Service
+- ✅ **Local AES fallback**: Development mode using local encryption keys
```

Comment:
Not quite "development mode". We should skip encyrption for development. AES_KEY can work in production, it's a no-cost option that requires manual key rotation. (TBD on if that's preserved, looking now...)